recipes/apache-sedona/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "apache-sedona" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apache-sedona-{{ version }}.tar.gz
+  sha256: ae0078004b6bd3376d7b7369dc04fa39824b38faa41ffeee77afe03612b3872e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - attrs
+    - pyspark <3.1.0
```

Comment:
Just note that upstream has changed this so it will likely need an update with the next release:

https://github.com/apache/incubator-sedona/blob/eea460f332d78516ec61cd4b67e817f19b10bc53/python/setup.py#L40